### PR TITLE
[MOB-2994] deprecations

### PR DESCRIPTION
**Pull request by roninopf**
## 🔹 Jira Ticket(s)

* [MOB-2994](https://iterable.atlassian.net/browse/MOB-2994)

## ✏️ Description

* removes and/or replaces code we marked as deprecated now that we're releasing `6.3.0` versions

### CODE REVIEW

File: swift-sdk/Internal/LegacyRequestHandler.swift [hunk 1]
```diff
@@ -225,30 +225,6 @@ class LegacyRequestHandler: RequestHandlerProtocol {
...
-                        onSuccess: OnSuccessHandler? = nil,
-                        onFailure: OnFailureHandler? = nil) -> Future<SendRequestValue, SendRequestError> {
-        let result = apiClient.track(inAppOpen: messageId)
-        return applyCallbacks(successHandler: onSuccess,
-                              andFailureHandler: onFailure,
-                              withIdentifier: "trackInAppOpen",
-                              forResult: result)
-    }
-    
-    @discardableResult
```
Comment (Ayyanchira) [inline swift-sdk/Internal/LegacyRequestHandler.swift:241]: This one did not had //deprecation Mark... is this change intentional?

File: swift-sdk/Internal/OfflineRequestProcessor.swift [hunk 2]
```diff
@@ -213,37 +213,6 @@ struct OfflineRequestProcessor: RequestProcessorProtocol {
...
-        }
-
-        return sendIterableRequest(requestGenerator: requestGenerator,
-                                   successHandler: onSuccess,
-                                   failureHandler: onFailure,
-                                   identifier: #function)
-    }
-    
-    @discardableResult
-    func trackInAppClick(_ messageId: String,
```
Comment (Ayyanchira) [inline swift-sdk/Internal/OfflineRequestProcessor.swift:233]: same here

File: swift-sdk/Internal/OnlineRequestProcessor.swift [hunk 3]
```diff
@@ -219,30 +219,6 @@ struct OnlineRequestProcessor: RequestProcessorProtocol {
...
-                        onFailure: OnFailureHandler? = nil) -> Future<SendRequestValue, SendRequestError> {
-        let result = apiClient.track(inAppOpen: messageId)
-        return applyCallbacks(successHandler: onSuccess,
-                              andFailureHandler: onFailure,
-                              withIdentifier: "trackInAppOpen",
-                              forResult: result)
-    }
-    
-    @discardableResult
-    func trackInAppClick(_ messageId: String,
```
Comment (Ayyanchira) [inline swift-sdk/Internal/OnlineRequestProcessor.swift:236]: and here as well

Comment (Ayyanchira) [reply swift-sdk/Internal/OnlineRequestProcessor.swift:236]: I think open and click go together in these ones ... 👍

File: tests/ui-tests/UITests.swift [hunk 4]
```diff
@@ -101,43 +101,6 @@ class UITests: XCTestCase {
         waitForElementToAppear(app.staticTexts["MyUniqueCustomAction"])
     }
     
-    func testShowSystemNotification() {
```
Comment (Ayyanchira) [inline tests/ui-tests/UITests.swift:104]: Test should still pass right? Why to remove them?

File: swift-sdk/Internal/LegacyRequestHandler.swift (hunk 1 above)
Comment (roninopf) [reply swift-sdk/Internal/LegacyRequestHandler.swift:241]: We only marked deprecation on the very top levels, but these calls were exclusive to their path/stack.

File: tests/ui-tests/UITests.swift (hunk 4 above)
Comment (roninopf) [reply tests/ui-tests/UITests.swift:104]: It won't work, actually, the deprecated calls were tied into the Storyboard. But it's also not relevant to our SDK anymore.

**Review by Ayyanchira — APPROVED** [review]
